Snippets/Snippets_6/UnitTesting/AdditionalServices/MyMessage.cs

```diff
@@ -0,0 +1,6 @@
+namespace Snippets6.UnitTesting.AdditionalServices
+{
+    class MyMessage
+    {
+    }
+}
```

Comment:
Ah, you mean in the actual file, rather than rendered in docs? If so, sure, no problem!